### DISCUSSION TITLE
SVGs broken on readme.

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#1314] [Q&A]
No real explanation needed. Happened after updating to 3.30

![grafik](https://user-images.githubusercontent.com/11576465/207451514-45712659-9226-4692-a430-cec6d7e9d8a7.png)

**Comment by lowlighter** [comment]
Hi ! 

Sorry for the inconvenience, it may be a regression caused by #1293

Files stored in `.cache` seems to be fine, so it seems to only impact markdown templates when they try to include back the SVG cached links (the base64 version is supposed to be displayed in environments without cache like web instances)

I'll try to patch this in the following days 👍 
Thanks for reporting this issue
